partials/campaign_copy.html

```diff
@@ -1,31 +1,31 @@
 <div id="campaign_container">
    <div id="campaign_summary">
-      <h1>Clone "{{currentCampaign.title}}"</h1>
+      <h1>{{ts('Clone', {domain: 'de.systopia.campaign'})}} "{{currentCampaign.title}}"</h1>
```

Comment:
syntax `{ts}string{/ts}` doen't work in such angular templates. 
This version is ok?